server/src-lib/Hasura/RQL/DDL/Relationship/Types.hs

```diff
@@ -102,6 +105,19 @@ type ObjRelDef = RelDef ObjRelUsing
 
 type CreateObjRel = WithTable ObjRelDef
 
+data CreateRemoteRelationship =
+  CreateRemoteRelationship
+    { createRemoteRelationshipName :: RemoteRelationshipName
```

Comment:
As a convention we use the short prefix `crr` instead of `createRemoteRelationship` for brevity.

Reply:
It's a convention that I wouldn't recommend, brevity just means "unreadable" and I have autocomplete, but this is your codebase. :man_shrugging:

Reply:
Haha. I can understand your view. We have a coding guidelines coming up and we will make sure to consider this but till then let's stick with the present conventions.